x/ibc/24-host/validate.go

```diff
@@ -0,0 +1,102 @@
+package host
+
+import (
+	"regexp"
+	"strings"
+
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+)
+
+// ICS 024 Identifier and Path Validation Implementation
+//
+// This file defines ValidateFn to validate identifier and path strings
+// The spec for ICS 024 can be located here:
+// https://github.com/cosmos/ics/tree/master/spec/ics-024-host-requirements
+
+// regular expression to check string is lowercase alphabetic characters only
+var isAlphaLower = regexp.MustCompile(`^[a-z]+$`).MatchString
+
+// regular expression to check string is alphanumeric
+var isAlphaNumeric = regexp.MustCompile(`^[a-zA-Z0-9]+$`).MatchString
```

Comment:
We already have `IsAlphaNumeric` in `/types`. We should  move `isAlphaLower` there too.